packages/ui/react-ui-sfx/src/hooks/useAudioStream.ts

```
@@ -11,25 +11,33 @@ export type UseAudioStream = {
   const audioContextRef = useRef<AudioContext>(undefined);
   const analyserRef = useRef<AnalyserNode>(undefined);
   const dataArrayRef = useRef<Uint8Array>(undefined);
   const tracksRef = useRef<MediaStreamTrack[]>(undefined);
-  const sourceRef = useRef<MediaStreamAudioSourceNode>(undefined);
+  const sourceRef = useRef<AudioNode>(undefined);
 
   const close = () => {
-    log.info('closing microphone');
     sourceRef.current?.disconnect();
     sourceRef.current = undefined;
     tracksRef.current?.forEach((track) => track.stop());
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)